interceptor/opencensus/opencensus.go

```diff
@@ -106,7 +104,26 @@ func (oci *OCInterceptor) Export(tes agenttracepb.TraceService_ExportServer) err
 			return err
 		}
 
-		// Otherwise add these spans to the bundler
-		traceBundler.Add(recv.Spans, len(recv.Spans))
+		// As per https://github.com/census-instrumentation/opencensus-service/issues/43,
+		// we support multiplexing. Thus, if a Node hasn't been sent from downstream, assume
+		// that those spans are from the initiatingNode
```

Comment:
The comment says that if the Node is not sent assume spans are from the last received Node:
https://github.com/census-instrumentation/opencensus-proto/blob/master/src/opencensus/proto/agent/trace/v1/trace_service.proto#L68

Reply:
Great catch @bogdandrutu, I've updated the code with the logic to use the last received and non-nil node, if the current message's Node is nil. PTAL.